dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -144,6 +149,11 @@ export const Liveness = ({
 				insertNewBlocks(data.html);
 				hydrateBlocks();
 
+				if (lastUpdated) {
+					lastUpdated.setAttribute('dateTime', new Date().toString());
+					updateTimeElements();
```

Comment:
Yep, originally I left it to be updated automatically. Felt like it looked better if it was updated at the exact same moment that the new blocks get added to the page though.

Reply:
That's actually a really interesting point. I didn't see this before but the issue is the display string of the element will no longer reflect the `dateTime` attribute value and yeah, it will basically be wrong.

You're right. This string should be updated at this point.

Maybe there's a way to update the string value on the element directly (the same as you're updating the attribute) or maybe there's a way to change the api for `updateTimeElements` to make it more targetted? You could perhaps have `updateTimeElement` (singular) or perhaps pass in an element?